io.github.ripose.Memento.yml

```diff
@@ -0,0 +1,577 @@
+app-id: io.github.ripose.Memento
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: memento
+rename-desktop-file: memento.desktop
+rename-icon: memento
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --filesystem=home/.var/app/io.github.ripose.Memento/config/memento/scripts:create
+  - --filesystem=host:ro
+  - --env=LC_NUMERIC=C
```

Comment:
why?

Reply:
[mpv requires this](https://github.com/mpv-player/mpv/commit/1e70e82baa9193f6f027338b0fab0f5078971fbe)

Reply:
you mean mpv is so utterly broken that their maintainers use foul language to justify it?

Reply:
I think the bigger problem is with locales in general, but wm4 isn't exactly diplomatic about it.